AsyncTwitch/TwitchConnection.cs

```diff
@@ -55,20 +131,62 @@ private void Awake()
         public void StartConnection()
         {
             if (_isConnected) return;
-            
+
             _loginInfo = Config.LoadFromJSON();
             if (_loginInfo.Username == "") return;
             Connect("irc.twitch.tv", 6667);
             _isConnected = true;
         }
 
+        public void RegisterPlugin(string pluginIdentifier) {
```

Comment:
Every plugin should call StartConnection(), we should figure out a way to assign the identifies automagically. Possibly using the stack fram to see who's calling StartConnection() and Register Events.